charts/gateway/values.yaml

```diff
@@ -9,6 +9,13 @@ license:
   accept: false
   # existingSecretName: ssg-license
 
+# Use set diskless Configuration mode for Gateway
+# when true , get values from environment variable and on false will get values from node.properties
+disklessConfig:
+  enabled: true
+  value:
+  # existingSecretName:
```

Comment:
We can comment out value by default since this is only used when `enabled` is `false`.

Reply:
Addressed in https://github.com/CAAPIM/apim-charts/pull/318/commits/9b9d8d060847d479d6993d9698df8e6afd3cda32